subsys/bluetooth/audio/ascs.c

```diff
@@ -1518,6 +1518,49 @@ struct ascs_parse_result {
 	size_t count;
 };
 
+static bool ascs_valid_metadata_type(uint8_t type, uint8_t len)
+{
+	switch (type) {
+	case BT_AUDIO_METADATA_TYPE_PREF_CONTEXT:
+	case BT_AUDIO_METADATA_TYPE_STREAM_CONTEXT:
+		if (len != 2) {
```

Comment:
The lengths here are one less than in the assigned numbers, so I assume these are without the type octet.

Reply:
Correct, the length and type fields have already been extracted at this point.